src/symbols.js

```diff
@@ -19,7 +19,8 @@ export const ACTIVITY_STATUS_TIMED_OUT = Symbol('ACTIVITY_STATUS_TIMED_OUT')
 
 export const ACTIVITY_TYPE_TRANSACTION = Symbol('ACTIVITY_TYPE_TRANSACTION')
 
-export const TRANSACTION_STATUS_ERROR = Symbol('TRANSACTION_STATUS_ERROR')
+export const TRANSACTION_STATUS_UPCOMING = Symbol('TRANSACTION_STATUS_UPCOMING')
+export const TRANSACTION_STATUS_SIGNING = Symbol('TRANSACTION_STATUS_SIGNING')
 export const TRANSACTION_STATUS_PENDING = Symbol('TRANSACTION_STATUS_PENDING')
 export const TRANSACTION_STATUS_SUCCESS = Symbol('TRANSACTION_STATUS_SUCCESS')
-export const TRANSACTION_STATUS_UPCOMING = Symbol('TRANSACTION_STATUS_UPCOMING')
+export const TRANSACTION_STATUS_ERROR = Symbol('TRANSACTION_STATUS_ERROR')
```

Comment:
These symbols were alphabetically sorted, is there a good reason to change that sort?

Reply:
I think it makes sense to sort these symbols according to the order in which they occur in the transaction lifecycle. This helps understand what each of these symbols stands for:

1. Upcoming - Waiting for a previous transaction to finish
2. Signing - Waiting for the user to sign the transaction through the Web3 provider
3. Pending - Waiting for the transaction to be mined in the blockchain
4. Success - The transaction was mined
5. Error - There was an error